test/e2e/common_suite_test.go

```diff
@@ -521,3 +544,24 @@ func doTestCreatePeerPodAndCheckEnvVariableLogsWithImageAndDeployment(t *testing
 	expectedPodLogString := "ISPRODUCTION=true"
 	newTestCase(t, "EnvVariablePeerPodWithBoth", assert, "Peer pod with environmental variables has been created").withPod(pod).withExpectedPodLogString(expectedPodLogString).withCustomPodState(v1.PodSucceeded).run()
 }
+
+func doTestCreatePeerPodWithPVCAndCSIWrapper(t *testing.T, assert CloudAssert, myPVC *v1.PersistentVolumeClaim, pod *v1.Pod, mountPath string) {
+
+	testCommands := []testCommand{
+		{
+			command:       []string{"lsblk"},
+			containerName: pod.Spec.Containers[2].Name,
+			testCommandStdoutFn: func(stdout bytes.Buffer) bool {
+				if strings.Contains(stdout.String(), mountPath) {
+					log.Infof("PVC volume is mounted correctly: %s", stdout.String())
+					return true
+				} else {
+					log.Errorf("PVC volume is failed to mount target path: %s", stdout.String())
```

Comment:
Nit: PVC volume is failed to mount target path -> PVC volume failed to be mounted at target path?

Reply:
fixed.